### ISSUE TITLE
Spoppy crashes when trying to play a song

### ISSUE THREAD

**Issue by gunnardag** [sindrig/spoppy#159]
Spoppy crashes when trying to play a song from an album when in "search for album".

To recreate:

1.  **sa** _to search for album_
2. **random access** _because daft punk_
3. **1** _for Daft punk, random access memories_
4. **4** _to select the song Within from that album_
5. **pl** _to play the song Within from that album_

Crashlog: 
```
Traceback (most recent call last):
  File "/usr/local/bin/spoppy", line 11, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/__init__.py", line 69, in main
    navigator.start()
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 63, in start
    self.navigate_to(main_menu)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 100, in navigate_to
    self.navigate_to(response)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 100, in navigate_to
    self.navigate_to(response)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 100, in navigate_to
    self.navigate_to(response)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 100, in navigate_to
    self.navigate_to(response)
  File "/usr/local/lib/python2.7/dist-packages/spoppy/navigation.py", line 86, in navigate_to
    response = response()
  File "/usr/local/lib/python2.7/dist-packages/spoppy/menus.py", line 841, in replace_current
    self.playlist.tracks.results.index(self.track)
AttributeError: 'Sequence' object has no attribute 'results'
```

**Comment by sindrig** [comment]
@gunnardag can you verify this is fixed in 2.1.2?

**Comment by gunnardag** [comment]
This fixes the issue
